govcd/api_vcd_test.go

```diff
@@ -1033,6 +1033,24 @@ func (vcd *TestVCD) removeLeftoverEntities(entity CleanupEntity) {
 			vcd.infoCleanup(notDeletedMsg, entity.EntityType, entity.Name, err)
 		}
 		return
+	case "standaloneVm":
+		vm, err := vcd.vdc.QueryVmById(entity.Name) // The VM ID must be passed as Name
+		if IsNotFound(err) {
+			vcd.infoCleanup(notFoundMsg, entity.EntityType, entity.Name)
+			return
+		}
+		if err != nil {
+			vcd.infoCleanup("removeLeftoverEntries: [ERROR] retrieving standalone VM '%s''. %s\n",
+				entity.Name, err)
+			return
+		}
+		err = vm.Delete()
+		if err != nil {
+			vcd.infoCleanup("removeLeftoverEntries: [ERROR] Deleting VM '%s' : %s\n",
```

Comment:
Total nit, make it same case as above error?

```suggestion
			vcd.infoCleanup("removeLeftoverEntries: [ERROR] deleting VM '%s' : %s\n",
```

Reply:
changed